### DISCUSSION TITLE
Where's The Browserify Bundle?

### DISCUSSION THREAD

**Discussion by Devmond** [cloudscape-design/components#1978] [Q&A] [answered]
I see that there used to be a browser include version of the distribution, is that still being published?

I have ran into a wall trying to browserify, not being a node/babel expert I am sure it is my fault :)

If someone can point me to a version for including on the web or the magic browserify command that can bundle it, that'll be appreciated.

**Comment by just-boris** [comment, marked as answer]
Hello!

Having a build tool (webpack, or vite, or any other) is required to consume our components. We do not provide standalone bundles

**Reply by Devmond** [reply]
Thanks for the response @just-boris . Too bad, I am trying to create a basic low code implementation for Cloudscape as shown here: https://www.youtube.com/watch?v=nuA_QVVCFy8

You already have demos published, is there some place where the browser js that support the demos are published? I looked at the samples in the repo but don't see any clean browser js exports. I am a webpack/vite noob and trying to avoid acquiring that expertise right now :) Any help would be appreciated.

**Reply by just-boris** [reply]
Thanks for the context, but nothing changes in my reply. Using a bundler is a must, you cannot avoid that.
